rotkehlchen/tests/unit/decoders/test_sdai_gnosis.py

```diff
@@ -0,0 +1,245 @@
+import pytest
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.history.events.structures.evm_event import EvmEvent
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+from rotkehlchen.constants.assets import A_XDAI, A_SDAI
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.gnosis.modules.sdai.constants import CPT_SDAI
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('gnosis_accounts', [['0x78E87757861185Ec5e8C0EF6BF0C69Fa7832df6C']])
+def test_deposit_sdai(database, gnosis_inquirer, gnosis_accounts):
+    user_address = gnosis_accounts[0]
+    tx_hash = deserialize_evm_tx_hash('0x1342646cab122d58f0b7dfae404dad5235d42224de881099dc05e59477bb93aa')  # noqa: E501
+    actual_events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=gnosis_inquirer,
+        database=database,
+        tx_hash=tx_hash
+    )
+    location_label = b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00x\xe8wW\x86\x11\x85\xec^\x8c\x0e\xf6\xbf\x0ci\xfax2\xdfl'
+    timestamp = TimestampMS(1707169525000)
+    gas_amount = '0.000367251244452481'
+    deposit_amount = '315'
+    withdraw_amount = '303.052244055946806232'
```

Comment:
you can define all of them in one line

```suggestion
    gas_amount, deposit_amount = '0.000367251244452481', '315', '303.052244055946806232'
```